proxy/logging/LogField.cc

```diff
@@ -772,7 +772,9 @@ LogFieldList::marshal_len(LogAccess *lad)
   int bytes = 0;
   for (LogField *f = first(); f; f = next(f)) {
     if (f->type() != LogField::sINT) {
-      bytes += f->marshal_len(lad);
+      const int len = f->marshal_len(lad);
```

Comment:
It looks like you are referring to the ATS 5.3.x branch in your comment above.  This looks like it has been resolved on master and 7.1.x.  Please confirm.